cadasta/common/setting.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Cadasta project - **Setting utilities**
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+from qgis.PyQt.QtCore import QSettings
+
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__revision__ = '$Format:%H$'
+__date__ = '20/12/16'
+__copyright__ = 'Copyright 2016, Cadasta'
+
+default_domain = 'https://demo.cadasta.org/'
+
+
+def set_setting(key, value):
+    """ Set value to QSettings based on key.
+
+    :param key: unique key for setting.
+    :type key: QString
+
+    :param value: value to be saved.
+    :type value: QVariant
+
+    """
+    settings = QSettings()
+    settings.setValue(key, value)
+
+
+def delete_setting(key):
+    """ Delete setting from QSettings.
+
+    :param key: unique key for setting.
+    :type key: QString
+
+    """
+    settings = QSettings()
+    settings.remove(key)
```

Comment:
You have to add a prefix such as 'cadasta/' to the key. QSettings is shared among QGIS and every plugins so you have to add prefix.
In QGIS Desktop, then the QGIS preferences and then `advanced` you can see all the keys in the QSettings.